crates/iota-indexer/src/models/transactions.rs

```diff
@@ -40,6 +40,14 @@ pub struct TxInsertionOrder {
     pub tx_digest: Vec<u8>,
 }
 
+/// Struct for inserting values into `tx_insertion_order` without providing the
+/// `insertion_order` value, which is autogenerated by the database.
+#[derive(Clone, Debug, Insertable)]
+#[diesel(table_name = tx_insertion_order)]
+pub struct NewTxInsertionOrder {
+    pub tx_digest: Vec<u8>,
+}
+
```

Comment:
We can use the `#[diesel(skip_insertion)]` attribute on `TxInsertionOrder::insertion_order` field instead. I think we want this to be autogenerated eitherway no? 

https://docs.diesel.rs/master/diesel/prelude/derive.Insertable.html#optional-field-attributes

Reply:
Done 81d372e6058f7e8d6d58288e12f82b79703ef8e6